movieApp.js

```diff
@@ -1,204 +1,279 @@
   }
 
-  readMoreBtn.addEventListener("click", toggle);
-  return movieContainer;
-}
+  function createMoviesPage() {
+    nowStreaming.innerHTML = "";
+    movies.forEach((movie) => {
+      const movieElement = createMovieCard(movie);
+      nowStreaming.appendChild(movieElement);
+    });
+  }
+  createMoviesPage();
+  // JS 2
+  const searchBtn = document.getElementById("search-icon");
 
-function createMoviesPage() {
-  nowStreaming.innerHTML = "";
-  movies.forEach((movie) => {
-    const movieElement = createMovieCard(movie);
-    nowStreaming.appendChild(movieElement);
-  });
-}
-createMoviesPage();
-// JS 2
-const searchBtn = document.getElementById("search-icon");
-
-// function to search
-const inputField = document.getElementById("keyword-search");
-searchBtn.addEventListener("click", () => {
-  const inputStr = inputField.value.toLowerCase();
-  nowStreaming.innerHTML = "";
-  let found = false; // Flag to check if any movie is found
-
-  movies.forEach((movie) => {
-    if (
-      movie.title.toLowerCase().includes(inputStr) ||
-      movie.description.toLowerCase().includes(inputStr)
-    ) {
-      found = true;
-      const searchMovieElement = createMovieCard(movie);
-      nowStreaming.appendChild(searchMovieElement);
-    }
+  // function to search
+  const inputField = document.getElementById("keyword-search");
+  searchBtn.addEventListener("click", () => {
+    const inputStr = inputField.value.toLowerCase();
+    nowStreaming.innerHTML = "";
+    let found = false; // Flag to check if any movie is found
+
+    movies.forEach((movie) => {
+      if (
+        movie.title.toLowerCase().includes(inputStr) ||
+        movie.description.toLowerCase().includes(inputStr)
+      ) {
+        found = true;
+        const searchMovieElement = createMovieCard(movie);
+        nowStreaming.appendChild(searchMovieElement);
+      }
+    });
     if (!found) {
       nowStreaming.innerHTML = `<p>Not Found</p>`;
     }
   });
-});
-
-//
-// creating toggling in filter/shorting option
-const shorting = document.getElementById("shorting");
-const filters = document.getElementById("filters");
-shorting.addEventListener("mouseover", function () {
-  this.classList.toggle("active");
-  if (filters.style.display === "block") {
-    filters.style.display = "none";
-  } else {
-    filters.style.display = "block";
-  }
-});
-const newRelease = document.getElementById("newRelease");
-newRelease.addEventListener("click", function () {
-  nowStreaming.innerHTML = "";
-  movies.forEach((movie) => {
-    if (movie.movie_year >= 2023) {
-      const movieElement = createMovieCard(movie);
-      nowStreaming.appendChild(movieElement);
+
+  // creating toggling in filter/shorting option
+  const shorting = document.getElementById("shorting");
+  const filters = document.getElementById("filters");
+  shorting.addEventListener("mouseover", function () {
+    this.classList.toggle("active");
+    if (filters.style.display === "block") {
```

Comment:
You can simplify this into a single line :)

Reply:
I tried to simplify it using ternary operator, I hope I did it right.